recursion/recurse.rb

```diff
@@ -1,10 +1,32 @@
 def fact(n)
+  # base case: 
+  if n == 0 || n == 1
+    return 1
+  end
+    return n * fact(n - 1)
 end
 
 def fib(n)
+  # base cases:
+  if n == 0
+    return 0
```

Comment:
Ahhhh! But you put this here because otherwise fib(2) doesn't work!  fib(2) is actually your second base case, not fib(0).